app/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -57,6 +62,10 @@ open class HomeAssistantApplication : Application() {
             }
         )
 
+        runBlocking {
+            keyChainRepository.load(applicationContext)
+        }
```

Comment:
This is going to affect everyones startup by running this blocking.  This should be launched on an IO coroutine scope like we do for `initCrashReporting`

Reply:
This should be resolved